README.md

```diff
@@ -1,70 +1,104 @@
 
-- [Using GitOps to instantiate the App](https://docs.giantswarm.io/advanced/gitops/apps/)
-- [Using our web interface](https://docs.giantswarm.io/platform-overview/web-interface/app-platform/#installing-an-app).
-- By creating an [App resource](https://docs.giantswarm.io/use-the-api/management-api/crd/apps.application.giantswarm.io/) in the management cluster as explained in [Getting started with App Platform](https://docs.giantswarm.io/getting-started/app-platform/).
+You will also have to create an object storage, either by enabling `MinIO` in the values file (only working on vintage clusters) or by providing an access to an existing object storage solution you own.
+This can be done by creating a `Bucket` CR when the cluster is runing the [object-storage-operator](https://github.com/giantswarm/object-storage-operator).
 
 ## Configuring
 
 ### values.yaml
 
-**This is an example of a values file you could upload using our web interface.**
+This is an example of a values file for a CAPZ cluster (namely : `glippy`) you could upload using our web interface :
 
 ```yaml
-# values.yaml
-
+psp:
+  enabled: true
+ingress:
+  enabled: true
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-giantswarm
+    giantswarm.io/external-dns: managed
+  className: nginx
+  hosts:
+    - pyroscope.glippy.azuretest.gigantic.io
+  tls:
+    - hosts:
+        - pyroscope.glippy.azuretest.gigantic.io
+      secretName: pyroscope-ingress-cert
+pyroscope:
+  pyroscope:
+    config: |
+      storage:
+        backend: azure
+        azure:
+          account_name: "<azure_storage_account_name>"
+          account_key: "<azure_storage_account_key>"
+          container_name: "giantswarm-glippy-pyroscope"
 ```
 
 ### Sample App CR and ConfigMap for the management cluster
 
 If you have access to the Kubernetes API on the management cluster, you could create
 the App CR and ConfigMap directly.
 
-Here is an example that would install the app to
-workload cluster `abc12`:
+Here is an example that would install the app in a vintage management cluster
 
 ```yaml
-# appCR.yaml
-
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  name: pyroscope
+  namespace: giantswarm
+spec:
+  catalog: giantswarm-playground-test
+  kubeConfig:
+    inCluster: true
+  userConfig:
+    configMap:
+      name: pyroscope-user-values
+      namespace: giantswarm
+  name: pyroscope
+  namespace: pyroscope
+  version: 0.1.0
 ```
 
-```yaml
-# user-values-configmap.yaml
+Note that if you want to deploy the app on a management cluster you will need to add the `app-operator.giantswarm.io/version: 0.0.0` label to the App CR's metadata so that the `app-operator` pick up and deploy the app.
```

Comment:
```suggestion
Note that if you want to deploy the app on a management cluster you will need to add the `app-operator.giantswarm.io/version: 0.0.0` label to the App CR's metadata so that the `app-operator` picks up and deploys the app.
```